test/ng/directive/ngModelOptionsSpec.js

```diff
@@ -449,7 +449,7 @@ describe('ngModelOptions', function() {
         });
 
 
-        it('should not trigger digest while debouncing', function() {
+        it('should not trigger digest while debouncing', inject(function($browser) {
```

Comment:
Because it was calling `$browser.defer.flush()` like the other tests (which is wrong in this case). I'll undo this...